src/components/signup.js

```diff
@@ -54,9 +54,207 @@ export default function Signup() {
   return (
     <>
       <CustomNavbar />
-      <Link to='create-user'>
-        <Button>Agree</Button>
-      </Link>
+      <div style={{padding: '3em'}}>
+        <div className='privacy-policy-title'>English</div>
+        <div dangerouslySetInnerHTML={{ __html: privacyPolicyEn }} className='privacy-policy-content'/>
+        <br/>
+        <div className='privacy-policy-title'>Korean</div>
+        <div dangerouslySetInnerHTML={{ __html: privacyPolicyKr }} className='privacy-policy-content'/>
+        <br/>
+        <Link to='create-user'>
+          <Button>Agree</Button>
+        </Link>
+      </div>
     </>
   );
 }
+
+
+const privacyPolicyEn = `
```

Comment:
https://stackoverflow.com/questions/50792942/how-to-import-html-file-into-react-component-and-use-it-as-a-component
How about separating this into a separate html file? It is too long.

Reply:
I tried. But it's not easy. I can use seperate js file instead. I will move them by another pr.